spec/factories/patients.rb

```diff
@@ -58,7 +58,8 @@
     parent_email do
       "#{parent_name.downcase.gsub(" ", ".")}#{random.rand(100)}@gmail.com"
     end
-    parent_phone { Faker::PhoneNumber.phone_number }
+    # Replace first two digits with 07 to make it a mobile number
+    parent_phone { Faker::PhoneNumber.phone_number.gsub(/\A\d{2}/, "07") }
```

Comment:
I think this does the same/similar (it's location aware)?
```suggestion
    parent_phone { Faker::PhoneNumber.cell_phone }
```